evidence-display/src/containers/realitio.js

```diff
@@ -77,31 +88,43 @@ class RealitioDisplayInterface extends Component {
     const questionID = arbitrationCreatedLogs[0].returnValues._questionID;
     const questionEventLog = await realitio.getPastEvents("LogNewQuestion", {
       filter: { question_id: questionID },
-      fromBlock: 0,
+      fromBlock: REALITY_STARTS_AT[realitioContractAddress.toLowerCase()] ?? 0,
```

Comment:
```suggestion
    fromBlock: Object.keys(REALITY_STARTS_AT).includes(realitioContractAddress.toLowerCase())
      ? REALITY_STARTS_AT[realitioContractAddress.toLowerCase()]
      : 0,
```

Reply:
shouldn't this also apply to the dynamic script?